spec/lib/efile/md/md502cr_calculator_spec.rb

```diff
@@ -330,39 +412,44 @@
       main_calculator.calculate
     end
 
-    context "when single filer" do
-      let(:filing_status) { "single" }
+    context "when filing_status is not mfj" do
+      %w[single married_filing_separately head_of_household qualifying_widow dependent].each do |filing_status|
```

Comment:
according to prod/program, "dependent" filing status should never be in a situation where they're calculating this value. However, just to be thorough, I'm going to include it to follow the rule